### DISCUSSION TITLE
How to add a prefix to all tables' names for tracking?

### DISCUSSION THREAD

**Discussion by yitistica** [mlflow/mlflow#7477] [Q&A] [answered]
meta:
mlflow version: 1.27

When I start a Mlflow server, i.e., using `mlflow server --backend-store-uri ... --default-artifact-root ... --host 0.0.0.0`, it automatically created the tables Mlflow needs for tracking, such as tables 'alembic_verison', 'experiments' table and so on. Is there a way to add prefix to all the tables generated by Mlflow (for project compliance reason)?

**Comment by WeichenXu123** [comment, marked as answer]
I don't think there's a way to do this. Could you file a feature request ticket ?
